packages/remark-cli/readme.md

```diff
@@ -7,39 +7,70 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-Command line interface for [**remark**][remark].
-
-*   Interface by [`unified-args`][unified-args]
-*   Loads [`remark-` plugins][plugins]
-*   Searches for [markdown extensions][markdown-extensions]
-*   Ignores paths found in [`.remarkignore` files][ignore-file]
-*   Loads configuration from [`.remarkrc`, `.remarkrc.js` files][config-file]
-*   Uses configuration from [`remarkConfig` fields in `package.json`
-    files][config-file]
+Command line interface to inspect and change markdown files with **[remark][]**.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [CLI](#cli)
+*   [Examples](#examples)
+    *   [Example: checking and formatting markdown on the CLI](#example-checking-and-formatting-markdown-on-the-cli)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [Sponsor](#sponsor)
+*   [License](#license)
+
+## What is this?
+
+This package is a command line interface (CLI) that you can use in your terminal
+or in npm scripts and the like to inspect and change markdown files.
+This CLI is built around remark, which is a very popular ecosystem of plugins
+that work with markdown as structured data: ASTs (short for abstract syntax
+trees).
```

Comment:
This same sentence is repeated in the monorepo too. Same suggestion there?

- The reason for the term first and explanation later is that I think more folks will have heard of `AST`. Similar to, say, CIA (...) or so. The full details are not required if the abbreviation is know, but can optionally enhance understanding
- The `as X using Y` to me doesn’t make a lot of sense because what I want to explain is that Y is a narrowed form of X. Similar to say, `as markup using markdown`. I want to say “specifically” or “otherwise known as”?

Reply:
ping @ChristianMurphy

Reply:
> Same suggestion there?

Same there

> The reason for the term first and explanation later is that I think more folks will have heard of AST

:thinking: alright

> I want to say “specifically” or “otherwise known as”?

Sure, how about:

```suggestion
that work with markdown as structured data, specifically ASTs (abstract syntax
trees).
```